app/support/payoff_manager.rb

```diff
@@ -0,0 +1,101 @@
+## Loan Manager to determine funding channel status
+## Determines using current month history and construction activity
+class PayoffManager
+  include Days360
+  class<<self
```

Comment:
indentation and spacing, also curious to hear what factors lead to decision on using singleton pattern?